x-pack/plugins/upgrade_assistant/public/components/tabs/checkup/deprecations/list.tsx

```diff
@@ -91,7 +88,7 @@ export const DeprecationList: StatelessComponent<{
     const indices = deprecations.map(dep => ({
       index: dep.index!,
       details: dep.details,
-      reindex: dep.message === OLD_INDEX_MESSAGE,
+      reindex: dep.reindex,
```

Comment:
I think this map is probably not necessary anymore, could probably just cast `deprecations` to `IndexDeprecationDetails[]`